content/zh/examples/controllers/nginx-deployment.yaml

```diff
@@ -16,6 +16,6 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx:1.15.4
```

Comment:
see https://github.com/kubernetes/website/blob/master/content/en/examples/controllers/nginx-deployment.yaml#L19

thanks for your contribution.
could you sync this to /en right now?
/hold

Reply:
For different languages, try to submit PR separately, thank you

Reply:
> could you sync this to /en right now?

OK!
I rebased commit to latest master.

> For different languages, try to submit PR separately

I dropped zh changes from this PR.

Thank you 😄

Reply:
@cakecatz
Thanks
/hold cancel